src/uikit.cs

```diff
@@ -12970,9 +12970,9 @@ interface UIStringAttributeKey {
 		[iOS (7, 0)]
 		[Internal, Field ("NSDocumentTypeDocumentAttribute")]
 		NSString NSDocumentTypeDocumentAttribute { get; }
-	//
```

Comment:
Not sure how this edit came about :(

Reply:
That is the auto format tool :)

Reply:
No worries this is the result of a missmatch, there is another PR in progress https://github.com/xamarin/xamarin-macios/pull/16971 that fixes this

Reply:
This has been fixed now.